src/dev-starter.ts

```diff
@@ -66,14 +71,15 @@ export class DevStarter {
       console.error(chalk.red(data.toString()));
     });
 
-    return new Promise((resolve) => {
+    return new Promise<{ launchUrl: string; localhostUrl: string }>((resolve) => {
       let projectRunning = false;
       serverProcess.on('error', (err) => {
         console.error(err);
       });
 
-      serverProcess.stdout.on('data', (data) => {
-        const output = data.toString();
+      serverProcess.stdout.on('data', async (data) => {
```

Comment:
I think this doesn't need to be `async` anymore:

```suggestion
      serverProcess.stdout.on('data', (data) => {
```